corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -1032,7 +1032,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
         } else {
             if (self.isLabel) {
                 self.controlWidth = "";
-                self.labelWidth = "";
+                self.labelWidth = constants.FULL_WIDTH;
```

Comment:
`col-12` to get the col padding you also get for any other fields that use `col-4` for the label and `col-8` for the widget.